Stripe/STPAddCardViewController.m

```diff
@@ -71,10 +71,10 @@ @interface STPAddCardViewController ()<STPPaymentCardTextFieldDelegate, STPAddre
 @end
 
 static NSString *const STPPaymentCardCellReuseIdentifier = @"STPPaymentCardCellReuseIdentifier";
-static NSInteger STPPaymentCardEmailSection = 0;
-static NSInteger STPPaymentCardNumberSection = 1;
-static NSInteger STPPaymentCardBillingAddressSection = 2;
-static NSInteger STPPaymentCardRememberMeSection = 3;
+static const NSInteger STPPaymentCardEmailSection = 0;
```

Comment:
Is there a reason these are not just an enum and instead are separate static consts?